nameable.rb

```diff
@@ -0,0 +1,5 @@
+module Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ ] Please from the project requirement, you should create a `Nameable` class, not a module so it would be better if you create a `Nameable` class instead of a module. :pray:

Reply:
> * [ ]  Please from the project requirement, you should create a `Nameable` class, not a module so it would be better if you create a `Nameable` class instead of a module. 🙏

- [ ] As suggested by the previous reviewer, kindly make a Nameable class, not a module, the requirement says class.